### ISSUE TITLE
SELECT without INTO

### ISSUE THREAD

**Issue by larshp** [abaplint/abaplint#1895]
TABLES or DATA missing, should give error, as the target is implicit
SELECT without INTO is also obsolete?

```abap
SELECT * FROM foobar WHERE field = 'bar'.
ENDSELECT.
```
